eng/common/scripts/Package-Properties.ps1

```diff
@@ -105,6 +105,20 @@ function Get-PkgProperties
     return $null
 }
 
+function Get-PrPkgProperties([string]$InputDiffJson) {
+    $pkgPropsResult = @()
+
+    Write-Host "In Get-PRPkgProperties function"
+
+    if ($GetPRPackageInfoFromRepoFn -and (Test-Path "Function:$GetPRPackageInfoFromRepoFn"))
```

Comment:
What do you expect one of these functions to look like?

Reply:
I think I found your python version https://github.com/Azure/azure-sdk-for-python/commit/5cd960a276e2056c7215ac5a831c7177fc9a88e3

I guess I had a solution in my head where you would call `Get-AllPkgProperties` then use the file diff list to find all the packages that have files under their package directory and that would be the set of package infos we would write out and use for any other purposes.

Reply:
[Would look like this](https://github.com/Azure/azure-sdk-for-python/blob/6de9adc570ead012a3c81f67e2524e24893396b9/eng/scripts/Language-Settings.ps1#L37)